paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/settings/SupportedPaymentMethodsSettingsDefinition.kt

```diff
@@ -14,14 +14,16 @@ internal object SupportedPaymentMethodsSettingsDefinition :
     override val saveToSharedPreferences: Boolean = false
 
     override fun configure(value: List<String>?, checkoutRequestBuilder: CheckoutRequest.Builder) {
-        checkoutRequestBuilder.supportedPaymentMethods(value)
+        if (!value.isNullOrEmpty()) {
```

Comment:
This is somewhat unrelated, mostly just a bug fix. I can pull this into a separate PR if you want.